dask/dataframe/io/tests/test_parquet.py

```diff
@@ -2859,6 +2859,65 @@ def test_split_row_groups_int_aggregate_files(tmpdir, engine, split_row_groups):
     assert_eq(df, ddf2, check_index=False)
 
 
+@PYARROW_MARK
+@pytest.mark.parametrize(
+    "filters,op,length",
+    [
+        ([("c", "!=", "a")], lambda x: x[x["c"] != "a"], 13),
+        ([("c", "==", "a")], lambda x: x[x["c"] == "a"], 2),
+    ],
+)
+@pytest.mark.parametrize("split_row_groups", [True, False])
+def test_filter_nulls(tmpdir, filters, op, length, split_row_groups, engine):
+    if engine == "pyarrow" and pa_version < parse_version("6.0.0"):
+        pytest.skip("PyArrow>=6.0.0 needed for null filtering")
+
+    path = tmpdir.join("test.parquet")
+    pd.DataFrame(
+        {
+            "a": [1, None] * 5 + [None] * 5,
+            "b": np.arange(14).tolist() + [None],
+            "c": ["a", None] * 2 + [None] * 11,
+        }
+    ).to_parquet(path, engine="pyarrow", row_group_size=10)
+
+    result = dd.read_parquet(
+        path,
+        engine=engine,
+        filters=filters,
+        split_row_groups=split_row_groups,
+    )
+    assert len(op(result)) == length
```

Comment:
Might also be worth checking that the result returned by dask matches the expected result

Reply:
(Similar comment for `test_filter_isna` below)